### PR TITLE
NCDA for CHW - tracked pregnancy logic

### PR DESCRIPTION
**Pull request by anvmn**
#767

### CODE REVIEW

**Comment by anvmn** [comment]
2 ANC encounters:

Peek 2023-07-04 23-10

**Comment by anvmn** [comment]
NCDA filled previously:

Peek 2023-07-04 23-09

File: client/src/elm/Measurement/View.elm [hunk 1]
```diff
@@ -2683,34 +2684,41 @@ resolveNCDAFormStep historyData form =
...
+
 showNCDAQuestionsByNewbornExam : Maybe PregnancySummaryValue -> Bool
 showNCDAQuestionsByNewbornExam newbornExamPregnancySummary =
     -- Verify that NCDA related questions were not answered at Neborn exam.
     -- This can happen, because needed questions were added after
     -- Newborn exam was launched, so, it could have been filled
     -- without them.
     -- It's enough to check if one of the questions was answered,
-    -- because both answres are required to save the form.
+    -- because both answereds are required to save the form.
```
Comment (mariano-dagostino) [inline client/src/elm/Measurement/View.elm]:
```suggestion
    -- because both answers are required to save the form.
```

**Review by mariano-dagostino — APPROVED** [review]
Looks good. One small typo.

**Pull request merged**